crates/wasmtime/src/component/instance.rs

```diff
@@ -520,6 +522,7 @@ impl<'a> Instantiator<'a> {
 pub struct InstancePre<T> {
     component: Component,
     imports: Arc<PrimaryMap<RuntimeImportIndex, RuntimeImport>>,
+    resource_imports: Arc<Vec<Option<RuntimeImportIndex>>>,
```

Comment:
This was a small enough change, that I just did it in this PR, but it did require a small addition to `PrimaryMap` impl to avoid iteration for constructing it